src/realm/decimal128.hpp

```diff
@@ -123,12 +123,17 @@ class Decimal128 {
     void unpack(Bid128& coefficient, int& exponent, bool& sign) const noexcept;
 
 private:
+    static constexpr int DECIMAL_EXPONENT_BIAS_128 = 6176;
+    static constexpr uint64_t MASK_SIGN = 0x8000000000000000ull;
```

Comment:
Where did these values come from and why are we confident they are correct vs the old ones?

Reply:
They come from the Intel library source.

Reply:
weren't the old ones also from the library source? Also, I see other values such as `49` used in several places and I have no idea why those specific values are used. We could create some constants for those too, adding comments to what they mean.

Reply:
@danieltabacaru I have now tried to explain the constants better